com.trekglobal.idempiere.rest.api/src/com/trekglobal/idempiere/rest/api/json/filter/DefaultQueryConverter.java

```diff
@@ -211,6 +225,8 @@ private String convertBinaryOperator(String left, String operator, String right)
 				}
 			} else {
 				// Check Right is Column
+				if (view != null)
+					right = toColumnName(view, right.trim());
```

Comment:
This is not taking into consideration when the right operand is not a column. For instance: 

/api/v1/models/bpartner?$filter=isEmployee eq true

returns 400 with the following error:
 _"detail": "Invalid column for filter: true"_

Reply:
ok, fix push.